packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_fr_FR.json

```
@@ -4,5 +4,7 @@
   "pci_databases_general_information_add_node_cancel": "Annuler",
   "pci_databases_general_information_add_node_description": "Vous disposez actuellement de {{nodesCount}} nœud(s). Voulez-vous vraiment en ajouter un ? Votre facture sera augmentée de",
   "pci_databases_general_information_add_node_success": "Votre demande d’ajout de nœud a bien été prise en compte et sera effective dans quelques minutes.",
-  "pci_databases_general_information_add_node_error": "Une erreur est survenue lors d'ajout de nœud : {{ message }}"
+  "pci_databases_general_information_add_node_error": "Une erreur est survenue lors d'ajout de nœud : {{ message }}",
+  "pci_databases_general_information_price_unit_month": "/ mois",
```

```suggestion
  "pci_databases_general_information_price_unit_month": "/mois",
```
French translation should not display "spaces" around slashes.